httpe/httpe.go

```diff
@@ -59,6 +59,25 @@ func NewHandlerFunc(h HandlerFuncE, ew ErrWriterFunc) http.HandlerFunc { //nolin
 	return NewHandler(h, ew).(http.HandlerFunc)
 }
 
+// NewDefaultHandler creates a new http.HandlerFunc which calls
+// HandlerE.ServeHTTP and if it returns an error calls WriteHTTPeErr
+// to create the appropriate response.
+func NewDefaultHandler(h HandlerE) http.Handler {
```

Comment:
Not sure about "Default" in the name, but I can't think of anything better at the moment. The problem is that there is nothing "default" about the handler. It is about the error writer. But I have nothing better.

Reply:
yeah, I agree, but I have no better suggestion either.
The only thing that comes to mind is to call this:
`NewHandler` and the other one `NewHandlerWithErrWriter` - what do you think?

Reply:
waiting for your thoughts.

Reply:
I thought of that alternative too, but it's a bit of a mouthful. Perhaps `WriteHTTPeErr()` needs to be renamed - what if it was called `WriteSafeErr()` and this was called `NewSafeHandler()` - "safe" may not be the best word - but that sort of change binds the two functions together. ("safe" because it doesnt leak internal information). Other words instead of safe: redacted, ... that's all i've got.

Reply:
ok. imo this is not better, but it isn't worth either.
i will change it.

Reply:
if you don't think it's better, then keep what you have. please.

Reply:
actually now that I have done it I do feel it is better....